tests/pyreverse/test_utils.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2021 Mark Byrne <mbyrnepr2@gmail.com>
+# Copyright (c) 2021 Andreas Finkler <andi.finkler@gmail.com>
+
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/master/LICENSE
+
+"""Tests for pylint.pyreverse.utils"""
+
```

Comment:
Those tests were initially inside ``test_writer.py``, but I think they belong in an own module.